lib/import/colorectal/providers/st_george/st_george_handler_colorectal.rb

```diff
@@ -0,0 +1,317 @@
+require 'possibly'
+require 'pry'
+require 'date'
+
+module Import
+  module Colorectal
+    module Providers
+      module StGeorge
+        # Process St George-specific record details into generalized internal genotype format
+        class StGeorgeHandlerColorectal < Import::Germline::ProviderHandler
+          include Import::Helpers::Colorectal::Providers::Rj7::Constants
+
+          def process_fields(record)
```

Comment:
I can't see any test coverage for `process_fields`?

Reply:
test added in  [b64fd4c](https://github.com/NHSDigital/data_management_system/commit/b64fd4c1b851661d098425a1ab1df681db92063f)